src/eei.cpp

```diff
@@ -57,6 +57,83 @@ Literal EthereumInterface::callImport(Import *import, LiteralList& arguments) {
       return Literal();
     }
 
+    if (import->base == Name("getBalance")) {
+      std::cout << "getbalance\n";
+
+      uint32_t addressOffset = arguments[0].geti32();
+      uint32_t resultOffset = arguments[1].geti32();
+
+      evm_address *address = new evm_address;
+      evm_uint256be *result = new evm_uint256be;
+
+      loadUint160(addressOffset, address);
```

Comment:
No need for new/delete, just place them on the stack. With other changes applied it should simplify to

```
evm_address address = loadUint160(addressOffset);
context.fn_table->get_balance(result, &context, &address);
```